src/explore-education-statistics-admin/src/pages/admin-dashboard/components/ManagePublicationsAndReleasesTab.tsx

```diff
@@ -166,7 +175,7 @@ const ManagePublicationsAndReleasesTab = ({
         });
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do we need to be ignoring the next line?

Reply:
Looks like someone else has taken care of this one.